acme/acme/client.py

```diff
@@ -18,7 +18,7 @@
 from acme import jose
 from acme import jws
 from acme import messages
-
+from certbot import log
```

Comment:
We don't want to create a dependency from `acme` to `certbot`. That will create a dependency loop, where each depends on the other. Also, we want to keep the dependencies for `acme` relatively small.

Instead of calling `log.except_hook`, my recommendation is to move the matching code you implemented over there into this file. In other words, once you're done you'll wind up with no diffs in `certbot`, only in `acme`. You'll also want to remove the `prints` and raise a new exception with the message you want, e.g. `raise Exception("foo ...")`.